governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+## ***DRAFT for Public Comment***
+
+# Technical Steering Committee (TSC) Charter
+
+## Section 1. Guiding Principle.  
+
+The Node.js Foundation will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 2. Evolution of Node.js Foundation Governance.
+
+Most large, complex open source communities have both a business and a technical governance model. Node.js Foundation’s technical leadership contains both a Technical Steering Committee (“TSC”) and Maintainers for major components or subsystems. Node.js Foundation’s business leadership is instantiated in a Board of Directors (the “Board”).  
+
+This Technical Steering Committee Charter reflects a carefully constructed balanced role for the TSC and the Board in the governance of Node.js Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the Board. The Board may additionally make amendments to the TSC charter at any time, though the Board will not interfere with day-to-day discussions, votes or meetings of the TSC.
+
+## Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.  
+
+The Board will set the overall TSC Policy. The policy will describe the overarching scope of the Node.js Foundation initiative, Node.js Foundation’s technical vision and direction and project release expectations in the form of expected cadence and intent. The Board will use the TSC as a delegate body for governing technical implementation, individual project scope and direction while they remain within the scope and direction of the policies as described in the TSC Policy document and approved by the Board.
```

Comment:
@mikeal This is a wonderful explanation. I was concerned myself, but it seems the concerns about the Board being able to change the TSC policy come from the mental model of viewing the Board and TSC as adversaries. As you point out, that's a flawed way of looking at it; if we ever reach that point, it's likely a symptom of deeper issues.

Is it worth some thought about what happens if there _is_ a disagreement between the TSC and Board? At least in my experience, business interests and technical interests don't always coincide; in your mind, how would such a conflict be resolved?